Makefile

```diff
@@ -9,17 +9,11 @@ LDFLAGS += -X "github.com/mattermost/mattermost-marketplace/internal/api.buildHa
 LDFLAGS += -X "main.upstreamURL=$(BUILD_UPSTREAM_URL)"
 SLS_STAGE ?= "dev"
 
+$(shell cp plugins.json ./cmd/lambda/)
```

Comment:
Given that there is no path traversal we need to copy the file into `cmd/lambda/`